giskard/integrations/wandb/wandb_utils.py

```diff
@@ -9,16 +8,19 @@
 
 try:
     import wandb  # noqa
-    from wandb.wandb_run import Run
 except ImportError as e:
     raise GiskardImportError("wandb") from e
 
 
-@contextlib.contextmanager
-def wandb_run(project: Optional[str] = None, **kwargs) -> Generator[Run, None, None]:
-    wandb.run = wandb.run or wandb.init(project=project or "giskard", **kwargs)
-    wandb.run._label(repo="Giskard")  # noqa
-    yield wandb.run
+def get_wandb_run(run):
+    if run is None:
+        if wandb.run is None:
+            raise ValueError("There are currently no active wandb runs available. Please follow the following "
+                             "documentation: https://docs.wandb.ai/ref/python/run to initiate a run. Once initiated, "
+                             "you can choose to pass is as follows 'to_wandb(run)', if not we will retrieve internally "
+                             "the 'wandb.run'.")
+        run = wandb.run
+    return run
```

Comment:
Can we simplify this?
```py
if run is not None:
    return run  # should we do other checks here?

if wandb.run is not None:
    return wandb.run

raise ValueError(...)
```

Also, should we use signature `def get_wandb_run(run=None)`?